### PR TITLE
improve instructions

### PR DESCRIPTION
**Pull request by choldgraf**
@henchc and I took a shot at setting up a binderhub and some things were confusing...this PR addresses a few points of confusion we had. Should still add some more to it...I'll remove the WIP when it's ready.

Addresses #112 and #111

### CODE REVIEW

**Comment by choldgraf** [comment]
some more edits added in there. when @minrk is done making changes to the binderhub app, let's take another pass at the instructions to make sure they're still correct. In the meantime we can either merge this and open a new PR, or wrap that work up into this PR

File: doc/setup-binderhub.rst [hunk 1]
```diff
@@ -1,40 +1,56 @@
 Set up BinderHub
 ================
 
+.. note::
+
+   Currently BinderHub requires that you configure the DNS of a web
+   address to point to the BinderHub IP address. This means that you need to
+   have control over a URL such as ``www.mydomain.com``.
```
Comment (minrk) [inline doc/setup-binderhub.rst]: Specifically, it's the use of `binderhub-deploy/support` to setup letsencrypt that has this requirement. If these docs update to only cover using the binderhub helm-chart directly (resolving #114), then this requirement goes away as well.

But presumably we also want to cover how to enable LEGO as well, at least at some point.

**Comment by choldgraf** [comment]
@minrk now that those big binderhub PRs are merged, should I change anything in this PR to update instructions?

**Comment by minrk** [comment]
Yes, I think so.

There user-facing changes, I think:

- helm-chart and binderhub-deploy repos don't need to be cloned. They can be fetched with helm:

        helm repo add jupyterhub https://jupyterhub.github.io/helm-chart

- installing the binderhub chart should now be similar to jupyterhub itself in zero-to-jupyterhub:

        helm install --name binder jupyterhub/binderhub --version v0.1.0-464c4f5

- in addition to the existing `proxy.secretToken` and `hub.cookieSecret`, there is one more required secret: `hub.services.binder.apiToken`. It can be generated in the same way as the others.

This could be a good opportunity to scrub the `support` instructions, which confuse things a bit, I think, but that can be another PR.

File: doc/setup-binderhub.rst (hunk 1 above)
Comment (choldgraf) [reply doc/setup-binderhub.rst]: now that binderhub can be installed with helm, can we edit this note to say this is only necessary w/ letsencrypt?

**Comment by choldgraf** [comment]
OK I'm gonna update this PR so we can merge now that we've a buncha new features in binderhub. I'll try adding in the comments you gave above Min, and will ping here if something doesn't work. Maybe this is something @yuvipanda and I can do together.

Anything else I should change per the new additions?

File: doc/setup-binderhub.rst (hunk 1 above)
Comment (minrk) [reply doc/setup-binderhub.rst]: yes

**Comment by minrk** [comment]
This is a big improvement, so I'll merge now despite the WIP. There's always future PRs!

**Pull request merged**